langserver/references.go

```diff
@@ -42,6 +42,8 @@ func (h *LangHandler) handleTextDocumentReferences(ctx context.Context, conn JSO
 	ctx, cancel := context.WithTimeout(ctx, documentReferencesTimeout)
 	defer cancel()
 
+	reverseImportGraphC := h.reverseImportGraph()
```

Comment:
As it stands it might not be obvious why this is here (at the top of the function) instead of at the bottom where it is used. Add a comment:

```
// Begin computing the reverse import graph immediately, as this occurs in the background
// and is IO-bound.
```